packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -533,4 +539,25 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
       };
     }
   }
+
+  private validateFieldMetadataInput<
+    T extends UpdateFieldInput | CreateFieldInput,
+  >(fieldMetadataInput: T): T {
+    if (fieldMetadataInput.name) {
+      try {
+        validateString(fieldMetadataInput.name);
+      } catch (error) {
+        if (error instanceof InvalidStringException) {
+          console.error(error.message);
```

Comment:
console logs are probably not needed

Reply:
otherwise if you want to provide logs, you can instantiate a logger in the class with "new Logger(Class.name)"

Reply:
Im good taking them of - initially it was because I thought I would provide less precise logs in the FE but actually they are virtually the same now.